code/modules/species/outsider/celestials.dm

```diff
@@ -101,3 +152,113 @@
 		/mob/living/carbon/human/proc/succubus_drain_finalize,
 		/mob/living/carbon/human/proc/succubus_drain_lethal,
 	)
+//Auril (and general flight) Abilities
+/datum/ability/species/toggle_flight
```

Comment:
all this ability stuff should probably go into code/modules/species/abilities.dm
besides the auril variant